lib/ui/window/window.cc

```diff
@@ -121,6 +121,30 @@ void Window::DispatchPointerPacket(const pointer::PointerPacketPtr& packet) {
   DartInvokeField(library_.value(), "_dispatchPointerPacket", {data_handle});
 }
 
+void Window::DispatchPointerDataPacket(const PointerDataPacket& packet) {
+  tonic::DartState* dart_state = library_.dart_state().get();
+  if (!dart_state)
+    return;
+  tonic::DartState::Scope scope(dart_state);
+
+  Dart_Handle data_handle =
+      Dart_NewTypedData(Dart_TypedData_kByteData, packet.data().size());
+  if (Dart_IsError(data_handle))
+    return;
+
+  Dart_TypedData_Type type;
+  void* data = nullptr;
+  intptr_t len = 0;
+  if (Dart_IsError(Dart_TypedDataAcquireData(data_handle, &type, &data, &len)))
+    return;
+
+  memcpy(data, packet.data().data(), packet.data().size());
+
+  Dart_TypedDataReleaseData(data_handle);
+  DartInvokeField(library_.value(), "_dispatchPointerDataPacket",
```

Comment:
This is a bit confusing to me. You released the underlying data of the `data_handle` in the line above. Then the handle is passed to `_disaptchPointerDataPacket`. Won't that data be garbage? Meanwhile the `data` and `len` acquired are never used. Isn't there some bounds checking that needs to be done?